docs/releasing/publishing.md

```diff
@@ -2,6 +2,11 @@
 
 Read the docs for [what to do before publishing a release](/docs/releasing/before-publishing-a-release.md) to ensure you are prepared to publish.
 
+See the [documentation on support branches](https://govuk-design-system-team-docs.netlify.app/how-we-work/version-control/support-branches.html#support-branches) if you need to:
+
+- publish a new release of previous major versions of GOV.UK Frontend
+- publish a ‘hotfix’ release of GOV.UK Frontend without including other unreleased changes on the main branch
```

Comment:
Want to format `main` as code to match https://github.com/alphagov/govuk-frontend/pull/4234 and https://github.com/alphagov/govuk-frontend/pull/4235?

```suggestion
- publish a ‘hotfix’ release of GOV.UK Frontend without including other unreleased changes on the `main` branch
```